selfdrive/ui/tests/test_translations.py

```diff
@@ -34,6 +34,7 @@ def test_missing_translation_files(self):
     assert os.path.exists(os.path.join(TRANSLATIONS_DIR, f"{self.file}.ts")), \
                     f"{self.name} has no XML translation file, run selfdrive/ui/update_translations.py"
 
+  @pytest.mark.serial
```

Comment:
Why doesn't `translations_dir` work to solve this? Is `TemporaryDirectory` somehow returning the same directory for two workers?

Reply:
This was removed since I used -n0 for the test_translation.py that make it serial.